services/src/api/handlers/wms.rs

```diff
@@ -351,6 +351,7 @@ async fn wms_map_handler<C: ApplicationContext>(
                 x_query_resolution,
                 y_query_resolution,
             ),
+            selection: Default::default(), // TODO
```

Comment:
do what?

Reply:
I made it more conrete e6b9479658da849934df78cc2b2b328aad6511b9